japan/assets/attendee-prerequisites.md

```diff
@@ -3,7 +3,7 @@
 
 # コミュニティ行動規範
 
-まずは、CNCF コミュニティ行動規範をご一読ください
+まずは、CNCF コミュニティ行動規範をご一読ください。
 https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/jp.md
```

Comment:
nit: How about write the URL on a new line?

Reply:
Rather, let's make it as a link. As same as this, do so for several URLs written in following sentences.

Reply:
I've updated commented items.

Reply:
Thanks for update.
/lgtm